{"Modified":"2022-09-29T17:15:00","Published":"2022-09-29T03:15:00","access":{},"assigner":"cve@mitre.org","cvss":null,"cwe":"Unknown","id":"CVE-2022-40278","impact":{},"last-modified":"2022-09-29T17:15:00","references":["https://github.com/Samsung/TizenRT/blob/f8f776dd183246ad8890422c1ee5e8f33ab2aaaf/external/iotivity/iotivity_1.2-rel/resource/csdk/security/provisioning/src/provisioningdatabasemanager.c#L103","https://github.com/Samsung/TizenRT/issues/5628","https://www.sqlite.org/c3ref/exec.html","https://github.com/Samsung/TizenRT/blob/f8f776dd183246ad8890422c1ee5e8f33ab2aaaf/external/iotivity/iotivity_1.2-rel/resource/csdk/security/provisioning/src/provisioningdatabasemanager.c#L107"],"summary":"An issue was discovered in Samsung TizenRT through 3.0_GBM (and 3.1_PRE). createDB in security/provisioning/src/provisioningdatabasemanager.c has a missing sqlite3_free after sqlite3_exec, leading to a denial of service.","vulnerable_configuration":[],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":[]}